.devops/azure-pipelines.test-pr-hotchocolate.yml

```diff
@@ -34,3 +34,16 @@ stages:
             inputs:
               artifactName: test-results
               pathtoPublish: "output/test-results"
+      - job: CheckPublicApi
```

Comment:
Lets spin this of in a GitHub action that is specifically for this task.

Reply:
I created one, but I can't seem to be able to test it with `act` locally. Is it possible to run the action in this PR already?